solutions/advanced/cf-891C

```diff
@@ -0,0 +1,152 @@
+---
+id: cf-891C
+source: CF
+title: Envy
+author: Chongtian Ma
+---
+
+[Official Analysis](https://codeforces.com/blog/entry/55841)
+
+## Explanation
+
```

Comment:
If an official editorial/solution exists you should provide a link

Reply:
?? its literally there